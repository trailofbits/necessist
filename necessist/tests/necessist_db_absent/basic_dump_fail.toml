args = ["--root=examples/basic", "--dump"]

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 1
