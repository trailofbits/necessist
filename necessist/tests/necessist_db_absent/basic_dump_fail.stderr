Error: --dump was passed, but no sqlite database found at "[CWD]/examples/basic/necessist.db"
